setup.cfg

```diff
@@ -1,2 +1,3 @@
 [flake8]
 max-line-length = 99
+ignore = E302
```

Comment:
just curious- is this because of the defining of regex's before functions?  I hadn't seen that pattern before